cmake/variables.cmake

```diff
@@ -5,6 +5,7 @@
 # This is to provide a user experience similar to find_package when
 # add_subdirectory or FetchContent is used to consume this project
 set(tomlplusplus_warning_guard "")
+option(tomlplusplus_CPP_INSTALL "Enable generation of tomlplusplus install targets" ${PROJECT_IS_TOP_LEVEL})
```

Comment:
Sorry, one more thing: remove the _CPP from the variable. "Plus plus" is already in the library name, and that suffix isn't used in the other variables. It's just noise.